### DISCUSSION TITLE
pcl如何检测有人私自公开分享了付费版的pcl？

### DISCUSSION THREAD

**Discussion by Deep-Dark-Forest** [Meloong-Git/PCL#1807] [提问] [answered]
正打算赞助。

**Comment by iPodtouch6557** [comment]
检测不到awa
曾经没赞助前我的PCL2.3.2由于系统误删丢了，新的版本也会被误删，就请求一个collaborator给我发了个snapshot版的2.3.2，没检测到……
但是如果相关信息被泄露龙猫还是可以及时阻止的（

**Comment by LTCatt** [comment, marked as answer]
有追查方法，但是不公开.jpg
